subtitles.go

```diff
@@ -248,6 +250,8 @@ type StyleAttributes struct {
 	WebVTTWidth          string
 }
 
+func (sa *StyleAttributes) propagateSRTAttributes() {}
```

Comment:
You need copy new `SRT*` attributes into proper fields such as corresponding `WebVTT*` fields. 

Same in `propagateWebVTTAttributes`, you need to do the reverse